pyhoma/client.py

```diff
@@ -206,7 +206,12 @@ async def check_response(response: ClientResponse) -> None:
         #  'error': 'Too many requests, try again later : login with xxx@xxx.tld'}
         #  'error': 'Bad credentials'}
         #  'error': 'Your setup cannot be accessed through this application'}
-        result = await response.json()
+        try:
+            result = await response.json()
+        except ContentTypeError:
+            body = await response.text()
+            raise Exception(body or "Something bad happened")
```

Comment:
For now raising an exception is fine, until we understand what the content is. Then I would like to handle it in a better way.

Can we change `Something bad happened` to a bit more descriptive error perhaps?

Reply:
What do you think about adding the URL, payload and status code?

Reply:
Agreed with a more explicit message